content/en/observability_pipelines/processors/remap_ocsf/custom_configuration.md

```diff
@@ -0,0 +1,369 @@
+| profiles | Yes      | An array of profile names that are added to the event class schema.                             |
+
+
+{{< tabs >}}
+{{% tab "YAML example" %}}
+
+```yaml
+version: 1
+metadata:
+  version: 1.1.0
+  class: Network Activity
+  profiles: [container, host, security_control]
+preprocess: []
+mapping: []
+
+```
+{{% /tab %}}
+{{% tab "JSON example" %}}
+
+```json
+{
+  "version": 1,
+  "metadata": {
+    "version": "1.1.0",
+    "class": "Network Activity",
+    "profiles": ["container", "host", "security_control"]
+  },
+  "preprocess": [],
+  "mapping": []
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+### Preprocessors
+
+Preprocessors rework the data to allow field-to-field mappings. Each entry in the `preprocess` section is an object consisting of a `function` name (required) and parameters associated with that function.
+
+#### `parse_csv`
+
+The `parse_csv` preprocessor:
+
+1. Extracts a `source` field.
+1. Parses `source` as CSV, maps the array of values to an object based on a list of field names in `columns`.
+1. Inserts it at `dest`.
+
+Fields with a corresponding `null` column name are dropped. One of the field names may have a wildcard `*` in which case it is assigned a string containing all the text from the fields that remain after those before and after have been mapped.
+
+{{< tabs >}}
+{{% tab "YAML example" %}}
+
+```yaml
+preprocess:
+- function: parse_csv
+  source: message
+  dest: .
+  skip_empty: true
+  - null
+  - receive_time
+  - serial
+  - type
+  - subtype
+  - null
+  - 'wildcard*'
+  - time_generated
+```
+
+{{% /tab %}}
+
+{{% tab "JSON example" %}}
+
+```json
+{
+  "preprocess": [
+    {
+      "function": "parse_csv",
+      "source": "message",
+      "dest": ".",
+      "skip_empty": true,
+      "columns": [
+        null,
+        "receive_time",
+        "serial",
+        "type",
+        "subtype",
+        null,
+        "wildcard*",
+        "time_generated"
+      ]
+    }
+  ]
+}
+```
+
+{{% /tab %}}
+{{< /tabs >}}
+
+
+In the examples above, the preprocessor expects an array object with seven or more columns. Since the first and sixth columns are `null`, they are dropped. The other six columns are mapped to:
+- `receive_time`
+- `serial`
+- `type`
+- `subtype`
+- `time_generated`
+- `wildcard`, which is set to the remaining columns of the array.
+
+### Mapping
+
+Each mapping entry consists of a `dest` path naming the destination field in the OCSF event and either a `source` path that names a field in the source event or a `value` that contains a literal constant to insert at that destination.
+
+A `source` path starting with `OCSF.` names a field in the output OCSF event that has been previously assigned.
+
+A `default` value may be named which is then used if the source paths do not exist before any further processing is done. The mapping may optionally also name a `function` to and/or a `lookup` table as described below.
```

Comment:
What does "named" mean here?

Reply:
Poor choice of words. That should say "specified".